### DISCUSSION TITLE
Bootstrap transition are not working on my computer..

### DISCUSSION THREAD

**Discussion by bumithaweerakkodi12** [twbs/bootstrap#39557] [Q&A] [answered]
I tried doing everything opening the webpage from other browsers in my computer, I cleared browser cache but nothing worked.
But When I open the webpage from another device it works properly..

Watch the video for get a better idea..*

https://youtu.be/IgBOxhYPeWQ

### Reduced test cases

T*

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Safari, Firefox, Microsoft Edge, Opera

### What version of Bootstrap are you using?

v.5.3*

**Comment by julien-deramond** [comment]
Hey @bumithaweerakkodi12 
I've converted your issue to a Q&A discussion first.
There's apparently an issue with the description as there's no video.

**Reply by bumithaweerakkodi12** [reply]
https://youtu.be/IgBOxhYPeWQ

this is the link Something happens When I try to upload the video

**Reply by julien-deramond** [reply, marked as answer]
Hmm, maybe you have a global accessibility setting that disables animations, that reduces motion?
Since you mentioned "Chrome, Safari, Firefox, Microsoft Edge, Opera", I suppose that it would be at the OS level.
For instance, if I enable "Reduce motion" on macOS, I have the same rendering as you.

**Reply by bumithaweerakkodi12** [reply]
Oh my god! Thank you for suggesting that It actually worked :)
